ba/lesson6/6_2/css/style.css

```diff
@@ -0,0 +1,104 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    transition: all .1s ease-in;
+  }
+  
+  header {
+    padding: 29px 0;
+    text-align: center;
+    color: #f66;
+  }
+  
+  .container {
+    width: 96%;
+    height: 96%;
```

Comment:
we do not need apply height for that element, we should apply height for small elements such as button or html, body { height: 100%}